proto/access.proto

```diff
@@ -79,6 +79,12 @@ message AccessAttempt {
   // Actor describes the entity attempting to gain access.
   // Optional.
   Actor actor = 3;
+
+  // The time the access attempt was made.
+  google.protobuf.Timestamp time = 4;
```

Comment:
Times should describe what they are, not just that they are a time: https://google.aip.dev/142#timestamps
For example `attempt_time` or `record_time`